src/lib/wallet/useTransferEvents.js

```diff
@@ -47,28 +48,27 @@ const useTransferEvents = () => {
       log.warn('updateAll failed', `Store failed with exception: ${message}`, exception)
       return
     }
-  }, [])
+  }, [getContext])
 
-  const initTransferEvents = useCallback(() => {
-    const lastBlock = parseInt(userStorage.userProperties.get('lastBlock') || 6400000)
-
-    log.debug('starting events listener', { lastBlock, subscribed: !!subscriptionRef.current })
+  const initTransferEvents = useCallback(async () => {
+    const subscribed = !!subscriptionRef.current
 
     if (subscriptionRef.current) {
+      log.debug('skipping', { subscribed })
       return
     }
 
-    goodWallet.watchEvents(lastBlock, toBlock => {
-      userStorage.userProperties.set('lastBlock', parseInt(toBlock))
-    })
+    await userStorage.ready
```

Comment:
is .ready where it initializes the properties? because we need to wait until userProperties is initliazed

Reply:
@sirpy fixed